### DISCUSSION TITLE
Disable Re-run jobs button

### DISCUSSION THREAD

**Discussion by torrocus** [community/community#28179] [Actions]
Is it possible to disable the "Re-run jobs" button for a selected workflow in GitHub Actions?

We have noticed that a re-run can lead to a situation where code older than HEAD is pushed onto production. Therefore, we would like to avoid this.

**Comment by alexjoeyyong** [comment]
I too would like to know an answer.

**Comment by alexjoeyyong** [comment]
@torrocus It's been over a year since you made this issue, so no idea if you need this anymore, but in the event someone else comes across this issue and is curious, here's a solution:

```bash
if [ "$GITHUB_RUN_ATTEMPT" -gt 1 ]; then
  echo "This is a rerun."
else
  echo "No re-runs for you. Go away."
  exit 1
fi

```
Add this to jobs you don't want the ability to re-run. Adjust this as needed. :) Not as ideal as a switch to turn this off across the board, but, this could work at least.

Example usage:
```yaml
no_reruns:
    runs-on: ubuntu-latest
    steps:
      - name: No reruns please
        run: |
         if [ "$GITHUB_RUN_ATTEMPT" -gt 1 ]; then
            echo "No re-runs for you. Go away."
            exit 1
          else
            echo "not a re-run"
          fi
```
